src/cms/config/functions/slugValidator.js

```diff
@@ -0,0 +1,6 @@
+module.exports = function (slug) {
+  const regex = new RegExp("^[a-z0-9\-\/]+(?:-[a-z0-9\-\/]+)*$|^$");
+    if (!regex.test(slug)) {
+      throw strapi.errors.badRequest('Please enter lower case letters, numbers, hyphens, or slashes for slug. No spaces.');
+    }
```

Comment:
can we have a second test to prevent consecutive forward slashes

e.g. 
```
  const regex2 = new RegExp(/^(?![\w/-]*\/\/)[\w/-]+$/gm);
  if (!regex2.test(slug)) {
    throw strapi.errors.badRequest('Consecutive forward slashes are not allowed in slugs.');
  }
```